test-five/infrastructure/lib/infrastructure-stack.ts

```diff
@@ -0,0 +1,196 @@
+import * as cdk from 'aws-cdk-lib';
+import {
+  aws_certificatemanager as acm,
+  aws_cloudfront as cloudfront,
+  aws_s3 as s3,
+  aws_s3_deployment as s3Deploy,
+  aws_route53 as route53,
+  aws_route53_targets as targets,
+  aws_cloudwatch as cloudwatch,
+  aws_cloudwatch_actions as cw_actions,
+  aws_sns as sns,
+  aws_sns_subscriptions as subscriptions,
+  CfnOutput,
+  CfnParameter,
+  RemovalPolicy
+} from 'aws-cdk-lib';
+
+// Setting the domain name.
+const DOMAIN_NAME = process.env.DOMAIN_NAME!;
+
+/**
+ * Represents the CloudFormation stack for the TestFive application.
+ * @class
+ * @extends cdk.Stack
+ */
+export class TestFiveStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    /**
+     * 👉 Environment Variable:
```

Comment:
emoji's in code comments haha what will they think of next

Reply:
I sometimes use emojis in pull request titles, scripts, or within the source code to emphasize specific points, but whether or not I do so depends on the project and the team's preferences...